test/cljs/status_im/test/utils/fx.cljs

```diff
@@ -17,6 +17,18 @@
   [{:keys [db]} b]
   (identity nil))
 
+(fx/defn hello4
+  {:doc "this function is useless as well"
+   :events [:test/valid1 :test/valid2]}
```

Comment:
@yenda  If i set multiple events like in this case, does it mean `hello4` would be called when `test/valid1`, `:test/valid2` gets dispatched? What is the use case for having multiple events handled by same defn?
Also, Is it possible to know what event triggered the call inside `hello4` body? Seems like event-name is not passed to the function.

Reply:
sure I will add that. 

The reason for supporting multiple events is to encourage the use of names that describe what happens for events and not actions. So the function can be an action and multiple events can trigger it. For instance navigate-back can be triggered by wallet/back-button-pressed and profile/back-button-pressed

Reply:
So I've looked into it and actually I think for the rare events that need to be self aware it is better to rely on an interceptor

Reply:
Ok, that makes sense not to add event-name to args as it's rarely used. It can be added to cofx by interceptor when needed as you said.